Keyboards/LanguageKeyboards/Hebrew/HEInterfaceVariables.swift

```diff
@@ -54,8 +35,8 @@ struct HebrewKeyboardProvider: KeyboardProviderProtocol {
     let keyboardBuilder = KeyboardBuilder()
       .addRow(["[", "]", "{", "}", "#", "%", "^", "*", "+", "="])
       .addRow(["_", "\\", "|", "~", "<", ">", "€", "£", "¥", "·"])
-      .addRow(["123", ".", ",", "?", "!", "'", "delete"])
-      .addRow(["ABC", "selectKeyboard", "space", "return"]) // "undo", "accent"
+      .addRow(["123", ".", ",", "?", "!", "'", "❌"])
+      .addRow(["אבג", "selectKeyboard", "רווח", "הכנס"])
```

Comment:
Looking at it, we have `space` and `return` in the Russian keyboard, so I think these should be reverted. Not ABC, that's correct, but the actual Hebrew for these keys will be substituted in later, or for return it similarly will be an icon like for `X` :)

Reply:
Will do